Dockerfile

```diff
@@ -0,0 +1,32 @@
+# base Docker image, remove "-alpine" if having trouble
+FROM node:10.5.0-alpine
+
+# added this step to prevent angular.json not found error
+WORKDIR /usr/src
+
+COPY . .
+
+# replace ng serve with ng serve --host 0.0.0.0 so that it serves outside
+# install jq for json edit
+RUN apk update \
+	&& apk add jq

```

Comment:
I believe we should not modify `package.json`, this seems not very efficient since we need to install an additional package. This could be achieved by `CMD npm start -- --host 0.0.0.0`.

Reply:
Ah, I see this makes more sense. I didn't know I could do that instead.